atlassian/test/cloud2.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "id": "9e1477fd-54ef-41fe-b747-bc9e6a11a925",
```

Comment:
This is what token introspeciton returns.
There are options of what container to choose. I employed the strategy of selecting container by workspace name. 
Id means CloudID.